pkgs/data/fonts/fira-math/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, fetchurl, stdenv }:
+
+let
+  pname = "fira-math";
+  version = "0.3.4";
+in
+stdenv.mkDerivation {
+  inherit pname version;
+
+  src = fetchurl {
+    url = "https://github.com/firamath/firamath/releases/download/v${version}/FiraMath-Regular.otf";
+    hash = "sha256-ICjL091NjAzxYIUg60dZlWqDpnkx17bY58MTUgGG41s=";
+    name = "FiraMath-Regular.otf";
+  };
+
+  dontUnpack = true;
+
+  installPhase = ''
+    install -D $src -t $out/share/fonts/opentype/
```

Comment:
This produces an output like `$out/share/fonts/opentype/3sdrv9f777vgl8989s7n3f3knsd8vjpx-FiraMath-Regular.otf`, probably not what we want.